README.md

```diff
@@ -126,8 +126,9 @@ To instrument an application running in Kubernetes, follow these steps:
        runAsUser: 0
        privileged: true
    ```
+2. Check if the configuration [shareProcessNamespace](https://kubernetes.io/docs/tasks/configure-pod-container/share-process-namespace/) is present in the pod spec, if not, please add it.
```

Comment:
Does this need to be added to the autoinstrumentation-go pod spec (above), just the application being run's pod spec, or both?

Reply:
Ah, pod level would mean it is for all running images:

https://github.com/open-telemetry/opentelemetry-go-instrumentation/blob/b1ee64b52e3a65c4535642f88e7575661c9e0134/.github/workflows/e2e/k8s/sample-job.yml#L14

We don't support a daemon set yet. Nevermind.